source/data/data_management_guidelines.rst

```diff
@@ -0,0 +1,159 @@
+
+This page is structured as follows:
+
+#. Choosing the right types of storage
+
+#. Tips for your data management plan
+
+#. Structuring your files and directories
+
+#. Data outflow to long-term storage
+
+#. Keeping track of your data
+
+
+Choosing the right types of storage
+===================================
+
+You will normally require at least the following two kinds of storage:
+
+- storage which is integrated in the HPC infrastructure
+- external storage: for data on which you are not actively computing anymore
+
+HPC storage
+-----------
+Carefully consider which type(s) of HPC storage are best suited for your work. The different VSC clusters all offer a :ref:`set of HPC storage locations<data location>`. For the Tier-2 clusters hosted at KU Leuven this is supplemented with so-called 'staging' storage. Staging uses the same (Lustre based) platform as the scratch storage and is also intended for larger amounts of data. Compared to scratch storage, however, staging is better suited for data which:
+
+- either needs to be shared by multiple users in a research project
+- and/or should not be subjected to automatic cleanup (files on scratch get deleted if they have not been accessed during the last month)
+
+Note that staging (even in the absence of automatic cleanup) should not be considered as a 'safe' storage location. Just like scratch, staging is single-copy and does not offer snapshots or backups. If somebody accidentally removes data or an unexpected system failure removes or corrupts data, this data is gone forever.
+
+Also keep in mind that staging is a paid service (charged based on peak usage per month) while temporarily increasing your scratch quotum can be done for free. If you only need more workspace for your own calculations, it may therefore be more advantageous to ask for a higher scratch quotum than to make more use of staging. In general, we recommend to only use staging storage if strictly required and to use scratch storage whenever possible. For all storage related requests, see also the `HPC Service Catalog <https://icts.kuleuven.be/sc/onderzoeksgegevens/english-version/HPC-storage>`_.
+
+External storage
+----------------
+As a general rule, data should be removed from HPC storage once it is no longer needed in further computations or postprocessing. Usually at least part of this data needs to be preserved for e.g. possible future reanalysis, sharing with other researchers, publication, et cetera. Such data then first needs to be copied to external storage (a process typically referred to as 'outflow'). Carefully consider your choice of external storage, in particular when it comes to (long-term) data safety, pricing, and how well it can be reached from within the HPC environment. Your options may include:
+
+- 'in-house' storage managed in your research group or department
```

Comment:
Since the page is for a specific cluster, but may be read by UHasselt and external users, let's maybe word it as follows:

```suggestion

- 'in-house' storage managed in your research group or department
- Tier-1 data
- Archive storage (K-drive) or large volume storage (L-drive) (KU Leuven specific)
- ManGO (KU Leuven specific) 
- Cold storage (like for example the upcoming platform `FriGO <https://rdm-docs.icts.kuleuven.be/frigo/index.html>`_, KU Leuven specific)
```

We could also link to the Tier-1 Data pages of the documentation, alas I cannot create that link in a suggestion :)